codec-http/src/main/java/io/netty/handler/codec/http/multipart/DeleteFileOnExitHook.java

```diff
@@ -71,10 +71,12 @@ public static boolean checkFileExist(String file) {
         return FILES.contains(file);
     }
 
-
-    static void runHooks() {
+    /**
+     * Clean up all the files.
+     */
+    private static void runHooks() {
         for (String filename : FILES) {
-            (new File(filename)).delete();
+            new File(filename).delete();
```

Comment:
Try catch ignore exception?

Reply:
If the file does not exist, return false .

Reply:
security manager still might throw an exception or NPE if null was passed as path.

Reply:
`ConcurrentHashMap` key not be null.  so avoided this problem